core/icons/comment_icon.ts

```diff
@@ -259,6 +303,12 @@ export class CommentIcon extends Icon implements IHasBubble, ISerializable {
     }
   }
 
+  onBubbleLocationChange(): void {
+    if (this.textInputBubble) {
```

Comment:
Is this one supposed to only care about the textInputBubble? Or should it also care about textBubbles? I think they're movable even in read-only mode. What are your thoughts on how we should handle that?

Reply:
I followed the model of `onSizeChange()`, which only concerns itself with the editable version. As you mentioned offline, presumably folks aren't serializing read-only workspaces, so I think this is fine to leave as-is.